index.html

```diff
@@ -9,27 +9,27 @@
   <body>
     <header class="header">
       <h1 class="header__heading">
-        <span class="header__heading1">the time is</span>
-        <span class="header__heading2">now</span>
+        <span>the time is</span>
+        <span class="header__heading_accent">now</span>
```

Comment:
It should be a dash, not an underscore in the second part :) `header__heading_accent`